.github/workflows/tests.yml

```diff
@@ -137,22 +135,15 @@ jobs:
             test-type: parallel
     timeout-minutes: 120
     env:
-      serial_or_parallel: ''
+      REPLAY_NAME: Win-${{ matrix.conda-version }}-Py${{ matrix.python-version }}
+      REPLAY_DIR: .\pytest-replay\
 
     steps:
       - name: Checkout repository
         uses: actions/checkout@v2
         with:
           fetch-depth: 0
 
-      - name: Set temp dirs correctly
-        shell: cmd
-        # https://github.com/actions/virtual-environments/issues/712
-        run: |
-          echo "TMPDIR=$env:USERPROFILE\AppData\Local\Temp" >> $env:GITHUB_ENV
-          echo "TEMP=$env:USERPROFILE\AppData\Local\Temp" >> $env:GITHUB_ENV
-          echo "TMP=$env:USERPROFILE\AppData\Local\Temp" >> $env:GITHUB_ENV
```

Comment:
is this no longer needed?

Reply:
Probably not, I tried to ignore the Windows job as I intended to review that in a followup PR